consensus/safety-rules/src/safety_rules.rs

```diff
@@ -77,6 +79,32 @@ impl SafetyRules {
             .map_err(|error| Error::InvalidAccumulatorExtension(error.to_string()))
     }
 
+    pub(crate) fn verify_order_vote_proposal(
```

Comment:
I wonder if these verifications are necessary. For order vote, validator only signs proposal that is certified, so it should already be verified. Doing these checks is for sure safe, but also wasting some cpu. WDYT? @zekun000

Reply:
I agree that the check may not be necessary as we implicitly call the function only on blocks with QC. But I prefer the safety rules to be complete.

Reply:
Removed some of the checks.